src/Configuration.php

```diff
@@ -53,6 +53,11 @@ class Configuration extends AbstractModel
      */
     public $group;
 
+    /**
+     * @var string
```

Comment:
Why not set typehints instead of php docs for new attributes/params?
```suggestion
     * @var string|null
```

Reply:
I already changed the phpdoc annotations to typehints